src/lib/structures/Command.ts

```diff
@@ -22,19 +22,19 @@ export default class Command {
         this.client = client;
         this.name = name;
         this.path = path;
-        this.aliases = (options && options.aliases) || [];
-        this.dm = (options && options.dm) || false;
+        this.aliases = (options?.aliases) ?? [];
+        this.dm = (options?.dm) ?? false;
         this.permission =
-            (options && options.permission) ||
+            (options?.permission) ??
             PermissionsLevels.SERVER_MEMBER;
-        this.mode = (options && options.mode) || Modes.FREE;
+        this.mode = (options?.mode) ?? Modes.FREE;
         this.access =
-            (options && options.access) || AccessLevels.DEFAULT;
-        this.ptb = (options && options.ptb) || false;
+            (options?.access) ?? AccessLevels.DEFAULT;
+        this.ptb = (options?.ptb) ?? false;
```

Comment:
Why are some of these on one line and others on two lines?

Reply:
@tjrgg seems like it just took the previous formatting and changed the code (eslint)